TW 337 Multiselect for collections (#4074)
postmanlabs/postman-docs: src/pages/docs/sending-requests/intro-to-collections.md

To move an existing request to a different collection:

1. Select the request you want to move.
1. Drag the request into the desired collection.

> Alternatively, open the request and select **Save As**, then choose the collection you want to move it to.

To move more than one request to a different collection:

> 1. Enter **⌘** or **Ctrl** on your keyboard, then select the requests you want to move.

Reviewer: ```suggestion
1. Press and hold **⌘** or **Ctrl**, then select the requests you want to move.
```

1. Select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> to the right of the collection name.
1. Select **Delete**.

> You can also select the collection you want to delete and enter **Delete** on your keyboard.

Reviewer: ```suggestion
You can also select the collection and press **Delete** on your keyboard.
```

Reviewer: Usually I don't think you need "on your keyboard" but it seems like it's needed here for clarity. Also another reason I don't like using "select" for keyboard keys as it could get really confusing. 🤪
